scripts/winesapos_convert.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Script by @GuestSneezeOSDev
+echo "Convering system to winesapOS in 3"
+sleep 1
+echo "Convering system to winesapOS in 2"
+sleep 1
+echo "Convering system to winesapOS in 1"
+sleep 1
+echo "System is converting ..."
+pacman -Sy --noconfirm
+useradd -m winesap
+echo "winesap:winesap" | chpasswd
+pacman -S git wget flatpak base-devel --noconfirm
+wget https://winesapos.lukeshort.cloud/repo/iso/winesapos-4.1.0/_test/winesapos-4.1.0-beta.1-minimal-rootfs.tar.zst
+pacman -U --noconfirm winesapos-4.1.0-beta.1-minimal-rootfs.tar.zst
+rm *.zst
+touch /etc/os-release-winesapos
+echo 'NAME="winesapOS"' >> /etc/os-release-winesapos
+echo 'PRETTY_NAME="winesapOS"' >> /etc/os-release-winesapos
+echo 'ID=winesapOS' >> /etc/os-release-winesapos
+echo 'ID_LIKE=arch' >> /etc/os-release-winesapos
+echo 'VERSION_ID=4.1.0-beta.3' >> /etc/os-release-winesapos
+echo 'HOME_URL="https://github.com/LukeShortCloud/winesapOS"' >> /etc/os-release-winesapos
+echo 'SUPPORT_URL="https://github.com/LukeShortCloud/winesapOS/issues"' >> /etc/os-release-winesapos
+echo 'BUG_REPORT_URL="https://github.com/LukeShortCloud/winesapOS/issues"' >> /etc/os-release-winesapos
+echo "[jupiter-staging]" >> /etc/pacman.conf
+echo "Server = https://steamdeck-packages.steamos.cloud/archlinux-mirror/\$repo/os/\$arch" >> /etc/pacman.conf
+echo "SigLevel = Never" >> /etc/pacman.conf
+echo "[winesapos]" >> /etc/pacman.conf
+echo "Server = https://winesapos.lukeshort.cloud/repo/winesapos/\$arch/" >> /etc/pacman.conf
+echo "SigLevel = Never" >> /etc/pacman.conf
```

Comment:
winesapOS does package signing. It's a bit of work for me to maintain but it ensures that if my packages ever get compromised, users will be notified.

You can remove the `SigLevel = Never` for winesapOS. Then you can add our GPG keys by running these commands:

https://github.com/LukeShortCloud/winesapOS/blob/4.1.0-beta.2/scripts/winesapos-install.sh#L296

After this line, you also need `pacman -S -y` to populate the repository information. Otherwise, Pacman will fail saying it does not have metadata for the wineaspOS repository.